corehq/apps/analytics/ab_tests.py

```diff
@@ -137,23 +137,3 @@ def context(self):
     'appcues_template_app_december2018',
     (APPCUES_TEMPLATE_APP_VARIANT_A, APPCUES_TEMPLATE_APP_VARIANT_B)
 )
-
-
-DEMO_WORKFLOW_HUBSPOT = 'hubspot'
```

Comment:
@gbova are we actually removing the demo workflow A/B? I just got a spec that still builds on this. @djmore9 @dimagi/product